apps/cli/src/store/build/actions.ts

```diff
@@ -0,0 +1,63 @@
+import { ISession } from '../../session/types';
```

Comment:
This doesn't quite feel like the right place for these functions? We are putting a bunch of non-store functionality under `store` (I'm already not super excited about all the stuff in `store/local/` - `actions.ts`, `notebook.ts`).

If I was looking around for these, I'd expect `checkLink` to be defined in `transforms/links.ts` (the only place it is used), and `addWarningToFile` to be in `utils` (it's pretty generic and used everywhere, plus we are already importing `store` in `utils`).

Reply:
moved.